Tests/ClientTests/Frontend/Home/TopSites/TopSitesDataAdaptorTests.swift

```diff
@@ -598,11 +596,7 @@ extension TopSitesDataAdaptorTests {
     }
 
     func add(searchEngine: OpenSearchEngine) {
-        mockSearchEngineProvider = MockSearchEngineProvider()
```

Comment:
I think we can remove the test variables `mockSearchEngineProvider` and `searchEngines` then?